backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxSparkPlanExecApi.scala

```diff
@@ -123,42 +123,44 @@ class VeloxSparkPlanExecApi extends SparkPlanExecApi {
       original)
   }
 
-  override def genTryAddTransformer(
+  override def genTryArithmeticTransformer(
       substraitExprName: String,
       left: ExpressionTransformer,
       right: ExpressionTransformer,
-      original: TryEval): ExpressionTransformer = {
+      original: TryEval,
+      funcName: String): ExpressionTransformer = {
```

Comment:
It is better to add a 'try_' prefix on substraitExprName if the try-function is needed to avoid introducing two functions names?
If we want to keep this implementation, could we rename funcName maybe like tryFunctionName and also document it to avoid misunderstanding? Thanks.

Reply:
Updated, thanks.